mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -129,17 +129,17 @@ CLASS net/minecraft/unmapped/C_xmldumst net/minecraft/world/GameRules
 		ARG 2 type
 	CLASS C_avotqrag Visitor
 		COMMENT A visitor used to visit all game rules.
-		METHOD m_hsodkmym visitBoolean (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
-			COMMENT Visit a boolean rule.
+		METHOD m_hsodkmym visitBooleanGameRule (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+			COMMENT Visit a boolean game rule.
 			COMMENT
 			COMMENT <p>Note {@link #visit(GameRules.Key, GameRules.Type)} will be called before this method.
 			ARG 1 key
 			ARG 2 type
-		METHOD m_mfrkzoox visit (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+		METHOD m_mfrkzoox visitGameRule (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
 			ARG 1 key
 			ARG 2 type
-		METHOD m_vtezodau visitInt (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
-			COMMENT Visit an integer rule.
+		METHOD m_vtezodau visitIntGameRule (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+			COMMENT Visit an integer game rule.
```

Comment:
I thought it was confusing that these are named `visitInt` and `visitBoolean` when they don't visit `int`s or `boolean`s but rather `IntGameRule`s and `BooleanGameRule`s.